Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -84,6 +84,7 @@
     'AP_RobotisServo',
     'AP_ToshibaCAN',
     'AP_NMEA_Output',
+    'UR_Atmosphere',
```

Comment:
It shouldn't be needed, as it lives in AP_Baro already

Reply:
in case that Atm could be used only on the frontend.